content/ai/concepts/neural-networks/terms/artificial-neural-networks/artificial-neural-networks.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Artificial Neural Networks'
+Description: 'A neural network is a method in artificial intelligence that teaches computers to process data in a way that is inspired by the human brain.'
+Subjects:
+  - 'Machine Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Machine Learning'
+  - 'ANN'
+  - 'Neural Networks'
+  - 'Weight & Bias'
+  - 'Brain Activity' 
```

Comment:
```suggestion
```
I don't think Brain Activity is a great tag, as it's only tangential to computer science.

Reply:
done